src/transform.ts

```diff
@@ -613,7 +668,7 @@ function mapKeys<TKey>(map: Map<TKey, any>): TKey[] {
 export function blockDataToBlockMetadata(blockData: BlockData): BlockMetadata {
   const { block, cert } = blockData
   return {
-    round: block.rnd,
+    round: BigInt(block.rnd),
```

Comment:
Here and a bunch of other places we've needed to cast to bigint, why is that? Is the raw block data not coming through as bigint? it should right?

Reply:
(or does it come in as `number | bigint`? In which case this makes sense...

Reply:
yes, the data returned from algod has type `number`.

Reply:
I think we should check this, `BlockData` is a type that we manage.